testing/scenario_app/compile_android_aot.sh

```diff
@@ -20,6 +20,10 @@ if [[ ! -d "$DEVICE_TOOLS" ]]; then
   exit 1
 fi
 
+echo "Using pub from $HOST_TOOLS/dart-sdk/bin/pub"
```

Comment:
nit: maybe run `pub --version` here just so we're sure of what version we're using?